far/findfile.cpp

```diff
@@ -509,9 +509,9 @@ void background_searcher::InitInFileSearch()
 	readBufferA.resize(readBufferSize);
 	readBuffer.resize(readBufferSize);
 
-	if (!SearchHex)
+	if (!Options.SearchHex)
 	{
-		m_TextSearcher = &init_searcher(m_TextSearchers, SearchCaseFold, strFindStr, false);
+		m_TextSearcher = &init_searcher(m_TextSearchers, Options.CaseSensitive, Options.Fuzzy, strFindStr, false);
```

Comment:
We can remove the [other](https://github.com/FarGroup/FarManager/blob/aeb8ffbbd83f3d620a601deb0a39aac6a653e2a0/far/string_utils.hpp#L237-L246) overload of `init_searcher` now, it's not used anywhere.

`search_case_fold` can be removed relatively easily too: it is only used in Viewer and fuzzy search is not supported there at all, so it is trivially reduceable to a boolean (up to you, I can address that separately later).

Reply:
I am about to add fuzzy search to Viewer; will remove `search_case_fold` and corresponding `init_searcher` overload with that change.

Any concerns about this?

Reply:
My apologies. I can see that Viewer is way more involved. I will try to do it, but it's gonna be a different story. For now, I removed both `search_case_fold` and `init_searcher` overload.

Reply:
Yes, Viewer is somewhat "special", but you are very welcome to try it :)